OKTA-554534 Updates to dev docs for event hook filtering (#4062)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

> Some installation notes:

Reviewer: What about just `Installation notes`, or just "Notes"

> * You don't need to have an account to install and run ngrok, but creating a free account provides more features including basic authentication.

Reviewer: Maybe something like:
```suggestion
* A free ngrok account isn't required, but can provide additional features including basic authentication.
* ```

> * You can install ngrok directly in your project folder, sample-app, as documented from the ngrok download page. Or you can install on your system's path directory to be able to run ngrok from any folder. Alternatively, you can install the executable in your favorite local folder, but you need the folder path when referencing the tool.

Reviewer: This seems a little clunky.

What about something like this:
`You can install ngrok in the system path directory or the project directory. Alternatively, you can install the executable in another local directory, as long as you reference the direcotyr path when using the tool.`

sample-app > ./ngrok http 8082
```

> Or if you installed in your system path:

Reviewer: Could we put these three options in a table instead? The formatting here seems funky.

Author: Removed two options. I think there's enough info here, and users can consult ngrok docs if they're having issues.

```shell
sample-app > ngrok http 8082
```

> Or if you used your favorite folder:

Reviewer: Something about favorite folder here seems off.

Could it be something like `Or if you used another directory`?

<div class="three-quarter border">

![An image of the ngrok inspection interface that includes details of a call.](/img/hooks/ngrok-and-event-hooks-web-interface.png)

</div>

## Create an Okta event hook

> Create the Okta event hook to work with your local application, which can now be exposed externally. The event hook must be set up and verified within your Admin Console.

Reviewer: ```suggestion
Create the Okta event hook to work with your local application, which can now be exposed externally. You must configure and verify the event hook within your Admin Console.
```

8. Click **Save & Continue**.

9. With your ngrok session and local application running, complete the one-time verification Okta call now. You can also verify the event hook later.

> ### Verify the event hook

Reviewer: This section is a little difficult to read with the nested sections and `Or:` Could you organize this a little differently?

Author: It's a little excessive and, really, not necessary based on the final step in the previous procedure. Endpoint verification is discussed elsewhere for background.

Verify the event hook to prove that your external service controls the endpoint. See [One-Time Verification Request](/docs/concepts/event-hooks/#one-time-verification-request).

To complete the one-time verification of the event hook:

* After creating the event hook, and:

  * Your local application server is running (`>node server.js`).

  * Your local ngrok session is running (and the forwarding URL is the same as the URL field defined in the event hook).

  Click **Verify** to complete the one-time verification step.

Or:

* After making sure that your ngrok session and local application are ready for the external verification call, go to the Event Hooks table, click the **Actions** dropdown menu of your **UNVERIFIED** event hook, and select **Verify**.

The event hook is now set up with a status of **VERIFIED** and is ready to send event hook calls to your external service.

> >**Note:** A successful event hook verification also indicates that your local application is useable with the ngrok session. Review the ngrok terminal or inspector interface for details on the first `GET` call to your local application.

Reviewer: ```suggestion
>**Note:** A successful event hook verification also indicates that your local application is usable with the ngrok session. Review the ngrok terminal or inspector interface for details on the first `GET` call to your local application.
```

Reviewer: Is this a Canada/US spelling difference?

Author: Not sure .... MMS uses your suggestion.

To run a preview call of your event hook:

1. In the Admin Console, go to **Workflow** > **Event Hooks**.
1. Locate the event hook that you created during the set-up step. In this example, select `New User Event Hook` or the name you gave the event hook.
1. Click the **Actions** menu for this hook, and select **Preview**.

> 1. In the **Configure Event Hook request** section, select an event from the **Event Type** dropdown menu. In this example, there’s only one: `User Created (user.lifecycle.create)`.

Reviewer: ```suggestion
1. In the **Configure Event Hook request** section, select an event from the **Event Type** dropdown menu. In this example, the only available option is: `User Created (user.lifecycle.create)`.
```
